src/content/docs/identityserver/upgrades/identityserver4-to-duende-identityserver-v7.mdx

```diff
@@ -0,0 +1,513 @@
+---
+title: "IdentityServer4 to Duende IdentityServer v7.2"
+sidebar:
+  order: 138
+  label: IdentityServer4 → v7.2
+---
+
+import { Badge } from "@astrojs/starlight/components";
+import { Code } from "astro/components";
+import { Steps } from "@astrojs/starlight/components";
+import { Tabs, TabItem } from "@astrojs/starlight/components";
+
+This upgrade guide covers upgrading from IdentityServer4 to Duende IdentityServer v7.2.
+IdentityServer4 reached its end of life (EOL) on December 13, 2022. It is strongly advised to migrate to Duende IdentityServer.
+
+Depending on your current version of IdentityServer4, different steps may be required.
+You can determine the version of IdentityServer4 by running the `dotnet list` command at the root of your project, or using NuGet tooling in Visual Studio or JetBrains Rider.
+
+<Tabs>
+  <TabItem label="Windows">
+    <Code
+      code={'dotnet list package | sls "IdentityServer4"'}
+      lang="bash"
+      title="Terminal"
+    />
+  </TabItem>
+  <TabItem label="macOS / Linux">
+    <Code
+      code={"dotnet list package | grep IdentityServer4"}
+      lang="bash"
+      title="Terminal"
+    />
+  </TabItem>
+</Tabs>
+
+This command will print a list of packages you are using in your solution, along with their version.
+
+```bash title="Output"
+   > IdentityServer4                            3.1.4       3.1.4
+   > IdentityServer4.EntityFramework            3.1.4       3.1.4
+   > IdentityServer4.AccessTokenValidation      3.0.1       3.0.1
+   > IdentityServer4.AccessTokenValidation      2.6.0       2.6.0
```

Comment:
Why is IdentityServer4.AccessTokenValidation listed here? Was the `dotnet list` command that produced this run at the solution level? Since we're looking for the IdSrv version I think we should remove these from the example. And maybe make it clear that `dotnet list` should be run in the project and not solution directory.